frontend/src/metabase/public/hooks/use-embed-frame-options.ts

```diff
@@ -4,7 +4,19 @@ import type { DashboardUrlHashOptions } from "metabase/dashboard/types";
 import { parseHashOptions } from "metabase/lib/browser";
 import { isWithinIframe } from "metabase/lib/dom";
 
-export const useEmbedFrameOptions = ({ location }: { location: Location }) => {
+import type { DisplayTheme } from "../lib/types";
+
+export const useEmbedFrameOptions = ({
```

Comment:
Require changes because some keys here e.g. `hide_parameters` has the wrong type. And that makes types return from this hook inside `frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.tsx` wrong.